src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseManagementForCaseWorkerService.java

```diff
@@ -318,4 +335,36 @@ private void sendUpdateSingleCaseECC(String authToken, CaseDetails currentCaseDe
         }
     }
 
+    public static void populateConciliationTrack(CaseData caseData) {
+        boolean isConTrackOp = false;
+        boolean isConTrackSt = false;
+        boolean isConTrackSh = false;
+        boolean isConTrackNo = false;
+
+        for (JurCodesTypeItem jurCodesTypeItem : caseData.getJurCodesCollection()) {
+            if (JUR_CODE_CONCILIATION_TRACK_OP.contains(jurCodesTypeItem.getValue().getJuridictionCodesList())) {
+                isConTrackOp = true;
+                break;
+            } else if (JUR_CODE_CONCILIATION_TRACK_ST.contains(jurCodesTypeItem.getValue().getJuridictionCodesList())) {
+                isConTrackSt = true;
+            } else if (!isConTrackSt
+                    && JUR_CODE_CONCILIATION_TRACK_SH.contains(jurCodesTypeItem.getValue().getJuridictionCodesList())) {
+                isConTrackSh = true;
+            } else if (!isConTrackSh && !isConTrackSt
+                    && JUR_CODE_CONCILIATION_TRACK_NO.contains(jurCodesTypeItem.getValue().getJuridictionCodesList())) {
+                isConTrackNo = true;
+            }
+        }
+
+        if (isConTrackOp) {
+            caseData.setConciliationTrack(CONCILIATION_TRACK_OPEN_TRACK);
+        } else if (isConTrackSt) {
+            caseData.setConciliationTrack(CONCILIATION_TRACK_STANDARD_TRACK);
+        } else if (isConTrackSh) {
+            caseData.setConciliationTrack(CONCILIATION_TRACK_FAST_TRACK);
+        } else if (isConTrackNo) {
+            caseData.setConciliationTrack(CONCILIATION_TRACK_NO_CONCILIATION);
+        }
```

Comment:
Should there be a default value set too? Also when jurisdictionCodesList is empty?